FlowersStore/Utils/Layout/UIView+Constraints.swift

```diff
@@ -0,0 +1,212 @@
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let constraint = trailingAnchor.constraint(equalTo: superview.trailingAnchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(equalTo height: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = heightAnchor.constraint(equalToConstant: height)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(equalTo nsLayoutDimension: NSLayoutDimension, multiplier: CGFloat = 1.0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let heightConstraint = heightAnchor.constraint(equalTo: nsLayoutDimension, multiplier: multiplier)
+        heightConstraint.priority = priority
+        heightConstraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(greaterThanOrEqualToConstant height: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = heightAnchor.constraint(greaterThanOrEqualToConstant: height)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(lessThanOrEqualToConstant height: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = heightAnchor.constraint(lessThanOrEqualToConstant: height)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func widthAnchor(equalTo width: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = widthAnchor.constraint(equalToConstant: width)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func widthAnchor(equalTo nsLayoutDimension: NSLayoutDimension, multiplier: CGFloat = 1.0) -> Self {
+        widthAnchor.constraint(equalTo: nsLayoutDimension, multiplier: multiplier).isActive = true
+        return self
+    }
+
+    @discardableResult func widthAnchor(greaterThanOrEqualToConstant width: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = widthAnchor.constraint(greaterThanOrEqualToConstant: width)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func widthAnchor(lessThanOrEqualToConstant width: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'widthAn...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Utils/Layout/UIView+Constraints.swift;
  Line: 185;
-->

- :warning: Function 'widthAnchor(lessThanOrEqualToConstant:priority:)' is unused